packages/wallet/src/createTransactionInternals.ts

```diff
@@ -36,6 +38,13 @@ export const createTransactionInternals = async (
   if (props.validityInterval.invalidBefore !== undefined) {
     body.set_validity_start_interval(props.validityInterval.invalidBefore);
   }
+  if (props.certificates?.length) {
```

Comment:
This actually impacts the minimum fee since it increases the tx size, so we need to think about how attaching certificates interoperates with input selection :thinking:

Reply:
Input selection constraints takes in `buildTx` and we're using `createTransactionInternals` for it in SingleAddressWalet,  so it already takes this into account for the minimum fee